core/templates/pages/exploration-editor-page/editor-tab/exploration-editor-tab.component.ts

```diff
@@ -313,10 +312,12 @@ angular.module('oppia').component('explorationEditorTab', {
                   contentId, stateName);
               }
             });
+            $rootScope.$applyAsync();
```

Comment:
Were there change detection issues? Just want to make sure that this is not introduced unnecessarily.

Reply:
actually, this is unnecessarily used forgotten to remove the code `Done`.